### PR TITLE
updated polymer-behaviors patch

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

File: patches/d2l-polymer-behaviors.patch [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
- import '../@polymer/polymer/polymer-legacy.js';
- import { dom } from '../@polymer/polymer/lib/legacy/polymer.dom.js';
--import { Element } from '../@polymer/polymer/polymer-element.js';
--import { useShadow } from '../@polymer/polymer/lib/utils/settings.js';
+@@ -1,78 +1,19 @@
+ import '@polymer/polymer/polymer-legacy.js';
+ import { dom } from '@polymer/polymer/lib/legacy/polymer.dom.js';
+-// WORKAROUND: polymer-modulizer grabs non-existing Element export from polymer-element
+-// TODO: Remove Element reference
+-import { PolymerElement as Element } from '@polymer/polymer/polymer-element.js';
```
Comment (svanherk) [inline patches/d2l-polymer-behaviors.patch]: So this is a part of the script that we might need to update

Comment (svanherk) [reply patches/d2l-polymer-behaviors.patch]: Here: https://github.com/Brightspace/polymer-3-converter/blob/master/toPolymer3.sh#L94
and here: https://github.com/Brightspace/polymer-3-converter/blob/master/toPolymer3.sh#L102
We keep the same number of `/..` combinations as before, should we just move this to only ever replace with `@polymer/polymer/polymer-element.js`?

Comment (dlockhart) [reply patches/d2l-polymer-behaviors.patch]: I think so yeah.

Comment (svanherk) [reply patches/d2l-polymer-behaviors.patch]: https://github.com/Brightspace/polymer-3-converter/pull/64

**Pull request merged**